projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/facet-list/facet-list.component.ts

```diff
@@ -94,6 +114,30 @@ export class FacetListComponent implements OnInit, OnDestroy {
     }
   }
 
+  ngAfterViewInit(): void {
+    this.renderFacets();
```

Comment:
This method of rendering is not friendly and is a bit of a workaround. It has issues not only with facet implementation but also with quick order and most likely other future implementations. We should move this change detection logic to solve the issue into the tab component.